content/en/blog/_posts/2021-09-20-admission-controllers-and-container-drift/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: blog
+title: "Using Admission Controllers to Detect Container Drift at Runtime"
+date: 2021-09-20
+slug: admission-controllers-and-container-drift
+---
+
+**Author:** Saifuding Diliyaer (Box)
+{{< figure src="intro-illustration.png" alt="Intro Illustration" width="70%" >}}
+*Illustration by Munire Aireti*
```

Comment:
nit: you can put the attribution into the figure

See https://gohugo.io/content-management/shortcodes/#figure for details.

Reply:
@sftim Thanks for the tip. Updated the PR with your suggestion.